gui/src/api/index.js

```diff
@@ -1,5 +1,5 @@
 import axios from 'axios';
 
 export const api = axios.create({
-  baseURL: 'http://127.0.0.1:8090/v1',
+  baseURL: 'http://localhost:8080/v1',
```

Comment:
So, it should be reverted ?

Reply:
I guess so. Hopefully it doesn't cause issues.